airflow-bi/pyproject.toml

```diff
@@ -14,6 +14,7 @@ boto3 = ">=1.0.0"
 s3fs = "^0.5.2"
 oauth2client = "^4.1.3"
 beautifulsoup4 = "^4.9.3"
+influxdb = "^1.8.10"
```

Comment:
why did you add this version? we are using a different one [here](https://github.com/pubnative/docker-images/blob/master/airflow/upgrade/Dockerfile#L10)